src/core/Debug.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  Disposable,
+  ExtensionMode,
+  Location,
+  TextEditorSelectionChangeEvent,
+  window,
+  workspace,
+} from "vscode";
+import { SyntaxNode, TreeCursor } from "web-tree-sitter";
+import { Graph } from "../typings/Types";
+
+const originalDebugLog = console.debug;
+const disabledDebugLog = () => {};
+
+export const log = {
```

Comment:
Why not just have this be an attribute of the debug graph component?  That seems more natural to me

Reply:
Because it's not available in every single file. Tying everything down to the graph structure is...cumbersome. But I will update it